vhdbuilder/packer/install-dependencies.sh

```diff
@@ -504,6 +504,25 @@ EOF
 
 fi
 
+if grep -q "GB200" <<< "$FEATURE_FLAGS"; then
+  # The GB200 feature flag should only be set for arm64 and Ubuntu 24.04, but validate
+  if [ ${UBUNTU_RELEASE} = "24.04" ] && [ ${CPU_ARCH} = "arm64" ]; then
+    # The open series driver is required for the GB200 platform. Dmesg output
+    # will appear directing the reader away from the proprietary driver. The GPUs
+    # are also not visible in nvidia-smi output with the proprietary drivers
+    apt install -y \
```

Comment:
we don't want to fix any versions here ? the repo only contains single versions ?

Reply:
we want the latest patch version based on what are in mlnx_ofed repo